config/prod/nf-syn23664726-s3.yaml

```diff
@@ -17,8 +17,8 @@ parameters:
   GrantAccess:
     - 'arn:aws:iam::325565585839:root'   # Required ARN for a synapse bucket
     - 'arn:aws:sts::526515999252:assumed-role/AWSReservedSSO_S3ExternalCollab_40c062f682e7f3f5/robert.allaway@sagebase.org'
+    - 'arn:aws:iam::694704239947:root' ## Nebula Genomics access
     - 'arn:aws:iam::694704239947:user/Manu' ## Nebula Genomics access
-    - 'arn:aws:organizations::694704239947:account/o-sa5l7ogl1k/694704239947' ## Nebula Genomics access
```

Comment:
i believe the same change caused the previous deployment to fail, https://github.com/Sage-Bionetworks/scicomp-provisioner/actions/runs/9746791529/job/26898038099#step:5:279

```
[2024-07-01 15:34:34] - prod/nf-syn23664726-s3 BucketPolicy AWS::S3::BucketPolicy UPDATE_FAILED Resource handler returned message: 
"Invalid principal in policy (Service: S3, Status Code: 400, Request ID: J2AJSD5JCF1XNFKE, 
Extended Request ID: Nq9qi8sdhNLIZh+J6D9qXTb/IFj4jbTGqm+dUHejovu0PBVGjVj+ZXghCh10UN3Kc/Xj6oix8es=)" 
(RequestToken: 180da11d-3b4e-a69d-5df2-93b5a8e5d6ef, HandlerErrorCode: GeneralServiceException)
```